### ISSUE TITLE
Implement settings.json for minor global settings

### ISSUE THREAD

**Issue by Hans5958** [arduano/Zenith-MIDI#15]
> I was thinking of adding a settings.json next to zenith for very minor global settings
> I want more meta settings rather than last state of the program
> options that should be there but are too minor to add to zenith's settings window 
> (like for example permanently ignoring kdmapi)

## Settings Idea

- Ignore KDMAPI
- Default settings window size
- Default plugin
- Default background

**Comment by javaarchive** [comment]
Agreed especially so I can finally run Zenith on my gaming computer(if I can't use zenith then the GPU is sort of wasted)

**Comment by arduano** [comment]
At the moment I'm slightly busy working on Kiva (which is the realtime midi player), but I will add this feature before the next 1.6 release.

Also dw, kiva will use much more gpu and multithreading than zenith does.

**Comment by Hans5958** [comment]
The feature is implemented in a future commit. This message will be edited when the commit is pushed.

Closed.
